backend/shork/src/main/kotlin/internal/program/AbstractProgram.kt

```diff
@@ -8,7 +8,7 @@ import software.shonk.interpreter.internal.util.CircularQueue
  * Abstract class representing a program that can be run on the Shonk interpreter It holds processes
```

Comment:
Shonk should be changed to Shork ;)

Reply:
True, but that wasn't me in this commit :3